consensus/vbft/service.go

```diff
@@ -304,14 +305,15 @@ func (self *Server) getChainConfig() (*vconfig.ChainConfig, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get peersinfo from leveldb: %s", err)
 	}
-	cfg, err := vconfig.GenesisChainConfig(config, peersinfo)
-	if err != nil {
-		return nil, fmt.Errorf("GenesisChainConfig failed: %s", err)
-	}
 	goverview, err := self.chainStore.GetGovernanceView()
```

Comment:
why GetGovernanceView() is a member function of chainstore? 
if has nothing with chainstore, it should be one util function.